src/script/components/ModalComponent.tsx

```diff
@@ -69,10 +69,8 @@ const ModalContentStyles: CSSObject = {
   flexDirection: 'column',
   fontSize: '0.875rem',
   margin: 'auto',
-  maxHeight: 615,
+  maxHeight: '90vh',
```

Comment:
Align with the user modal so that every modal is shown in its entirety even at high level of zoom